### PR TITLE
Remove \n from script

### PR DESCRIPTION
**Pull request by Thomas-Boi**
This should fix the issue #510

### CODE REVIEW

File: .github/workflows/post_peek_screenshot.yml [hunk 1]
```diff
@@ -47,7 +47,7 @@ jobs:
         if: success()
         env:
           OVERVIEW_IMG_MARKDOWN: ${{ fromJSON(steps.icons_overview_img_step.outputs.markdown_urls)[0] }}
-          DETAILED_IMGS_MARKDOWN: ${{ join(fromJSON(steps.icons_detailed_img_step.outputs.markdown_urls), '\n') }}
+          DETAILED_IMGS_MARKDOWN: ${{ join(fromJSON(steps.icons_detailed_img_step.outputs.markdown_urls), ' ') }}
```
Comment (Panquesito7) [inline .github/workflows/post_peek_screenshot.yml:50]: Why not just

```suggestion
          DETAILED_IMGS_MARKDOWN: ${{ join(fromJSON(steps.icons_detailed_img_step.outputs.markdown_urls), '') }}
```
?

**Review by Panquesito7 — DISMISSED** [review]
👍 apart from the above.

File: .github/workflows/post_peek_screenshot.yml (hunk 1 above)
Comment (Thomas-Boi) [reply .github/workflows/post_peek_screenshot.yml:50]: Oh, I was unsure what will happen if two image markdowns are next together. Now that you pointed it out, I just tested it and it seems like there's no issue.

**Review by Panquesito7 — APPROVED** [review]
Looks good. Can be merged. 👍

**Review by amacado — APPROVED** [review]

**Pull request merged**